### ISSUE TITLE
util module?

### ISSUE THREAD

**Issue by bitworm** [asmiller1989/Ethereum_Blockchain_Parser#5]
Hi,

I'm trying to run this, but `stream.py` can't find the `util` module.  Is this in the project somewhere or an external module?

```
File "stream.py", line 5, in <module>
    from Crawler import Crawler
  File "/Users/username/Ethereum_Blockchain_Parser/Preprocessing/Crawler/__init__.py", line 2, in <module>
    from util import *
ModuleNotFoundError: No module named 'util'
```

**Comment by N7Ren** [comment]
Hi,

I got it to run today without this error and there is no util.py in the project. Because of this I assume that it is an external dependency.

Did you install all required dependencies? Should be python 3.4 with the packages: contractmap, tqdm, requests, pymongo and graph-tool.

**Comment by N7Ren** [comment]
I just saw in Pull request #3 that the line is deleted from `__init__.py`. I guess then it is not needed at all.

**Comment by bitworm** [comment]
@rkernegger thank you for the info.  I can't get graph-tool working with python (anaconda distr., macos).  Can you provide install details/notes, and any other helpful notes?

**Comment by N7Ren** [comment]
@bitworm not really. I only installed it for Lubuntu with the instructions found here: https://git.skewed.de/count0/graph-tool/wikis/installation-instructions#debian-ubuntu and the installation worked fine. I haven't tried drawing any graphs though.
